core-plugins/docs/SSH-action.md

```diff
@@ -40,9 +40,9 @@ This example runs a script on demo@example.com:
     {
         "name": "SSHAction",
           "plugin": {
-            "name": "SSHAction",
+            "name": "SSH",
             "type": "action",
-            "label": "SSHAction",
+            "label": "SSH",
```

Comment:
we should remove this. its not used by the backend at all and is not in any of the other examples.